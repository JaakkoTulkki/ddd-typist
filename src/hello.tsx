import * as React from 'react';

export const Hello = () => <div>Hello chickens</div>;
