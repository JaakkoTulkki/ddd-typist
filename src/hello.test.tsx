import * as React from 'react';
import { Hello } from "./hello";
import { mount } from 'enzyme';

describe('hello', function () {
    it('should workd', function () {
        const component = mount(<Hello />);
        expect(component).toMatchSnapshot();
    });
});