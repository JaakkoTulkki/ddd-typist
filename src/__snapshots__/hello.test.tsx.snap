// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello should workd 1`] = `
ReactWrapper {
  "length": 1,
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": Object {
      "instance": <div>
        Hello chickens
      </div>,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Hello chickens",
      },
      "ref": null,
      "rendered": Array [
        "Hello chickens",
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": Object {
        "instance": <div>
          Hello chickens
        </div>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Hello chickens",
        },
        "ref": null,
        "rendered": Array [
          "Hello chickens",
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
